foia_hub/management/commands/load_agency_contacts.py

```diff
@@ -22,9 +15,33 @@ class Command(BaseCommand):
     """
 
     def handle(self, *args, **options):
-        try:
+        if len(args) > 0:
             yaml_folder = args[0]
-        except IndexError:
-            yaml_folder = _get_yaml_folder()
+        else:
+            yaml_folder = download_data()
 
+        print("Data directory: %s" % yaml_folder)
```

Comment:
Thanks @dlapiduz. Yeah, let's ignore my ramblings about writing to logs - but still use self.stdout.write() and self.stderr.write()